server/tournaments/index.js

```diff
@@ -320,25 +362,36 @@ class Tournament {
 			}
 		}
 	}
-	removeUser(user, output) {
-		if (!(user.userid in this.players)) {
-			output.sendReply('|tournament|error|UserNotAdded');
+
+	/**
+	 * @param {string} userid
+	 * @param {CommandContext} [output]
+	 */
+	removeUser(userid, output) {
```

Comment:
Why change this function to take a `userid` instead of a `User`?

Reply:
Banning an offline user with no object that's in the tour will try to remove them without a user object.